src/dcos_e2e/cluster.py

```diff
@@ -117,7 +120,6 @@ def from_nodes(
 
     @retry(
         exceptions=(subprocess.CalledProcessError),
-        tries=500,
```

Comment:
This is a behavioural change. Before - this timed out by default. Now, this does not time out unless the parent function is given a `timeout` value.

Reply:
Do you think this is undesirable? I think it adds consistency.